packages/nextjs/app/builders/0x1D714518c498246BA2e4f37805a3b4d8b98d9007/page.tsx

```diff
@@ -0,0 +1,86 @@
+import type { NextPage } from "next";
+import Image from "next/image";
+import Link from "next/link";
+import { FaLinkedin, FaGithub, FaGlobe, FaEnvelope, FaPhone } from "react-icons/fa";
+
+const PersonalPage: NextPage = () => {
+  return (
+    <div className="min-h-screen flex items-center justify-center bg-gradient-to-br from-gray-100 to-gray-200 dark:from-gray-800 dark:to-gray-900 py-8 px-4">
+      <div className="max-w-md w-full bg-white dark:bg-gray-800 shadow-xl rounded-2xl overflow-hidden">
+        <div className="flex flex-col items-center p-6">
+          <div className="w-32 h-32 relative rounded-full overflow-hidden border-4 border-blue-500">
+            <Image
+              src="/nft.jpg" 
+              alt="Avatar"
+              fill
+              className="object-cover"
+            />
```

Comment:
Maybe you can link your external NFT image directly instead of adding the asset to the project. If you don't have the URL, you can leave it as it is.